doc/manual/source/language/advanced-attributes.md

```diff
@@ -370,6 +370,18 @@ Derivations can declare some infrequently used optional attributes.
 
   ensures that the derivation can only be built on a machine with the `kvm` feature.
 
+- [`rejectSystemFeatures`]{#adv-attr-rejectSystemFeatures}\
+
+  If a derivation has the `rejectSystemFeatures` attribute, then Nix will only build it on a machine that does not has the corresponding features set in its [`system-features` configuration](@docroot@/command-ref/conf-file.md#conf-system-features).
```

Comment:
```suggestion
  If a derivation has the `rejectSystemFeatures` attribute, then Nix will only build it on a machine that does not have the corresponding features set in its [`system-features` configuration](@docroot@/command-ref/conf-file.md#conf-system-features).
```

Reply:
Thanks for catching that, it's fixed now.